docs/contributing-to-airbyte/building-new-connector/best-practices-for-connector-development.md

```diff
@@ -0,0 +1,53 @@
+# Best Practices for Connector Development
+
+In order to guarantee the highest quality for connectors, we've compiled the following best practices for connector development. Connectors which follow these best practices will be labelled as "Airbyte Certified" to indicate they've passed a high quality bar and will perform reliably in all production use cases. Following these guidelines is **not required** for your contribution to Airbyte to be accepted, as they add a barrier to entry for contribution. 
```

Comment:
I would add the same mention at the end (`but it doesn't hurt`)

Reply:
done